include/networkit/auxiliary/BucketPQ.hpp

```diff
@@ -82,6 +86,11 @@ class BucketPQ: public PrioQueue<int64_t, index> {
      */
     void insert(int64_t key, index value) override;
 
+    /**
+     * Returns the element on top of the PrioQ.
+     */
+    std::pair<int64_t, index> getMin();
```

Comment:
What about this signature?

Reply:
I added this to only get the first element without extracting it, this is consistent with `PrioQueue`.